server/controllers/TodoController.js

```diff
@@ -0,0 +1,193 @@
+const TodoModel = require('../models/TodoModel.js');
+const GroupModel = require('../models/GroupModel.js')
+const UserModel = require('../models/UserModel.js')
+const helpers = require('../helpers/helpers.js')
+const mongoose = require('mongoose')
+
+module.exports = {
+
+    changeStatus : function (req,res) {
+        TodoModel.findById(req.params.id)
+        .then((Todo) => {
```

Comment:
Kalau return sebuah `instance` bukan sebuah `class` lebih baik pakai huruf kecil

```suggestion
.then((todo) => {
  ```